articles/overview/overview.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Introduction to quantum computing
+description: Learn what quantum computing is, what quantum computers can do, and how you can learn quantum computing.
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.introduction
+---
+
+# Introduction to quantum computing 
+
+There are some problems so difficult, so incredibly vast, that even if every supercomputer in the world worked on the problem, it would still take longer than the lifetime of the universe to solve.
+
+Quantum computers hold the promise to solve some of our planet's biggest challenges - in environment, agriculture, health, energy, climate, materials science, and problems not yet even imagined. The impact of quantum computers will be far-reaching and have as great an impact as the creation of the transistor in 1947, which paved the way for today’s digital economy.
+
+## What is quantum computing?
+
+In modern usage, the word quantum means the _smallest possible discrete unit of any physical property_, usually referring to atomic or sub-atomic particles. Quantum computers use actual quantum particles – atoms or ions – as processing units, and are large, complex, and expensive devices. 
+
+Harnessing the unique behavior of quantum physics and applying it to computational computing, quantum computers introduce new concepts to traditional programming methods, making use of quantum physics behavior such as superposition, entanglement, and quantum interference. 
+
+## What can a quantum computer do?
+
+A quantum computer isn’t a supercomputer that can do everything faster, it just does some select things extremely well. 
+
+### Quantum simulation
+
+Since quantum computers use quantum particles as their processors, they are well-suited for modeling other quantum systems. Photosynthesis, superconductors, and complex molecules are examples of quantum systems that can be simulated using quantum programs.
```

Comment:
```suggestion
Since quantum computers use quantum phenomena in computation, they are well-suited for modeling other quantum systems. Photosynthesis, superconductivity, and complex molecular formation are examples of quantum mechanisms that can be simulated using quantum programs.
```